File: 18-03-2024_Level_Order_Traversal.py
class Solution:
    def levelOrder(self,root):
        queue = deque([root])
        res = []
        while queue:
            node = queue.popleft()
            res.append(node.data)
            if node.left:
                queue.append(node.left)
            if node.right:
                queue.append(node.right)
        return res

File: 18-03-2024_Level_Order_Traversal.java
class Solution
{
    static ArrayList <Integer> levelOrder(Node root) 
    {
        Queue<Node> q = new LinkedList<>();
            ArrayList<Integer> list = new ArrayList<>();
            q.add(root);
            while(!q.isEmpty())
            {
                Node r = q.remove();
                list.add(r.data);
                if(r.left != null)
                {
                    q.add(r.left);
                }
                if(r.right != null)
                {
                    q.add(r.right);
                }
            }
            return list;
    }
}
